experimenter/evaluate_pipeline_new.py

```diff
@@ -0,0 +1,199 @@
+import itertools as it
```

Comment:
Let's give this file a real name and, if needed, give the existing one an ugly name like `..._old.py`.